social_django/admin.py

```diff
@@ -54,9 +55,4 @@ class AssociationOption(admin.ModelAdmin):
     """Association options"""
     list_display = ('id', 'server_url', 'assoc_type')
     list_filter = ('assoc_type',)
-    search_fields = ('server_url',)
-
-
-admin.site.register(UserSocialAuth, UserSocialAuthOption)
-admin.site.register(Nonce, NonceOption)
-admin.site.register(Association, AssociationOption)
```

Comment:
If you reopen this file might be best to just comment them out and leave a comment explaining why.